test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -39,7 +39,7 @@ pick_a_version()
 {
     branch=$1
 
-    # Query out all released patch versions for a given release branch using "git tag"
+    # Query out all released git versions for a given release branch using "git tag"
```

Comment:
I believe the version this function queries out is a "mongo patch version" (e.g. mongodb-4.4.6) instead of a "git version".

Reply:
Made the change!